EnglishFilter/sections/content_blocker.txt

```diff
@@ -1323,6 +1323,10 @@ topbusiness24.ru,mydomsam.ru,hubuhu.ru,moysadinfo.ru,mybusinessplus.ru,dukand.ru
 freemc.host#@#.adsbygoogle
 ! https://github.com/AdguardTeam/AdguardFilters/issues/72650#issuecomment-771199740
 @@||googletagmanager.com/gtag/js$domain=exe.app
+! https://github.com/AdguardTeam/AdguardFilters/issues/77008
+@@||aiarticlespinner.co^$generichide
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=aiarticlespinner.co
+@@||pagead2.googlesyndication.com/pagead/js/r*/show_ads_impl.js$domain=aiarticlespinner.co
```

Comment:
Exchange your `@@||pagead2.googlesyndication.com/pagead/js/r*/show_ads_impl.js$domain=aiarticlespinner.co` to `@@||pagead2.googlesyndication.com/pagead/js/*/show_ads_impl$domain=aiarticlespinner.co`

Reply:
Okay.